src/Net/Http.fs

```diff
@@ -1100,6 +1105,18 @@ module private HttpHelpers =
         override x.GetResponseStream () = responseStream :> Stream
         member x.ResetResponseStream () = responseStream.Position <- 0L
 
+        member x.CharacterSet = httpProperty (fun r -> r.CharacterSet)
```

Comment:
Could you put a `///` comment on each of these?  I know these are missing on the rest but we may as well not make the problem worse :)